examples/azure_stt_example.py

```
@@ -0,0 +1,37 @@
+"""
+import sys
+
+import requests
+
+
+OVOS_HOST = "http://localhost:8080"
+
+
+def main(audio_path: str) -> None:
+    with open(audio_path, "rb") as fp:
+        r = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)